NitroxClient/NitroxClient.csproj

```diff
@@ -56,8 +56,23 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Communication\Client.cs" />
+    <Compile Include="Communication\Abstract\IClient.cs" />
+    <Compile Include="Communication\Packets\Processors\PlayerSlotReservationProcessor.cs" />
```

Comment:
Duh, my bad. I do the same thing all the time on large projects. I actually had a powershell script at one point in my life that did just that.

Reply:
We should set something up like that, I can potentially add it to one of the build scripts though VS may not like it as much. Maybe as git hook?

Reply:
We should set something up like that, I can potentially add it to one of the build scripts though VS may not like it as much. Maybe as git hook?

Reply:
We always roll it as a task in a build tool - generally psake for .NET project. That would allow us to do some really cool things from a local CI perspective as well, but I digress =)

Reply:
Afaik the maintainer @Sunrunner37 has set up a CI server somewhere...

Reply:
If the CI is heavy, then it may be beneficial to be able to run them locally rather than pound on an external service - but that may be reaching a little too far for this project.

Reply:
I was told a while ago that it did something, but I can't remember what. I think just some static code analysis for now.

Anyway, doing it locally and combining the sort with the commit one is about to make is a much better option IMHO.